javascript/ql/experimental/adaptivethreatmodeling/test/update_endpoint_test_files.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+# This script updates the JavaScript test data used by the endpoint CodeQL tests.
```

Comment:
How often should this script be run? How are we going to be reminded to run it? We _could_ create an action that does this on a schedule, but that might be overkill.

Reply:
Given that being out of date just means having potentially slightly worse test data, I don't think it's worth investing that much in automating it. I was thinking of simply setting up a Slack reminder.